src/library/dao/trackdao.cpp

```diff
@@ -1252,7 +1245,7 @@ bool setTrackAudioProperties(
             mixxx::audio::SampleRate(samplerate),
             mixxx::audio::Bitrate(bitrate),
             mixxx::Duration::fromSeconds(duration));
-    return false;
+    return false; // shouldDirty
```

Comment:
This was my attempt to document the meaning of this anonymous boolean which is otherwise hard to understand. 
This value is copied into the variable shouldDirty, found here: https://github.com/mixxxdj/mixxx/blob/94b94e4a350830fe3e13d88cc12e100c74480f78/src/library/dao/trackdao.cpp#L1517 
Do you have an idea to improve the situation?

Reply:
Mhmm, that makes assumptions about the calling code. In most cases where a boolean is too meaningless, you can always just introduce an enum with two members. Then you can deduce the meaning from the name of the return type and you can deduce the meaning of the value from their names. 
Naming is hard so I struggle to come up with a concrete suggestion myself. Though the problem primarily stems from the fact that I struggle to understand the purpose of `setTrackBeats`/`setTrackAudioProperties`/etc. Maybe you can think of something?

Reply:
I was about to fix it, but than I realized that all the code is not required. 
This was only used in setTrackKey() to replace a missing source flag with USER.
Even if the database is from an earlier version of Mixxx the data is consistent and the track can imported every time again.